compiler/rustc_infer/src/infer/mod.rs

```diff
@@ -508,7 +508,10 @@ pub enum RegionVariableOrigin {
     Coercion(Span),
 
     /// Region variables created as the values for early-bound regions.
-    EarlyBoundRegion(Span, Symbol),
+    ///
+    /// FIXME(@lcnr): This can also store a `DefId`, similar to
+    /// `TypeVariableOriginKind::TypeParameterDefinition`.
+    RegionParameterDefinition(Span, Symbol),
```

Comment:
is this used for late and early bound parameters?

Reply:
only early bound, copied he name from `TypeVariableOriginKind`